registry/registry_test.go

```diff
@@ -0,0 +1,57 @@
+package registry
+
+import (
+	"golang.org/x/net/context"
+	"testing"
+)
+
+func buildAnonymousDockerHub(t *testing.T, ctx context.Context) *Registry {
+	r, err := New(ctx, "https://registry-1.docker.io/", "", "")
+	if err != nil {
+		t.Error(err)
+	}
+	return r
+}
+
+func TestTags(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
There's really no reason to use `WithCancel`, other the fact that I'm using to this pattern and just wrote that without thinking...